cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -1975,6 +1975,11 @@ void serialize_to_hnswlib(
  * @note: When device memory is sufficient, the dataset attached to the returned index is allocated
  * in device memory by default; otherwise, host memory is used automatically.
  *
+ * @note: The `merge_params.merge_strategy` parameter cannot be set to MERGE_STRATEGY_LOGICAL
+ * using this API. Please use the API in the `cuvs/neighbors/composite/merge.hpp` instead
```

Comment:
i'm hesitant to point users towards thi API because IMO it shouldn't exist in the public APIs at all. The composite index should be fairly opaque to the user and we should be able to polymorpically return it instead of another index type (and have `merge()` coded to return "some" index, whether it be a cagra index or a composite index of cagra indexes. I think you are working towards this but want to make it concrete.

Reply:
I've removed the reference in the last commit